core/templates/dev/head/error/disabled_exploration.html

```diff
@@ -3,6 +3,7 @@
 {% block navbar_breadcrumb %}
 {% endblock navbar_breadcrumb %}
 
+<!-- TODO(mili) Translate the title. To avoid FOUC it must be done in the js -->
```

Comment:
That sounds reasonable, and is at least consistent for all pages, which is nice. I'm in favour.